package.json

```diff
@@ -42,6 +42,15 @@
                     "type": "boolean",
                     "default": true,
                     "description": "Set to `false` to disable `Ctrl+<key>` keybindings."
+                },
+                "amVim.defaultMode": {
+                    "type": "string",
+                    "default": "normal",
+                    "description": "default mode",

```

Comment:
Since initial mode can only be "Normal" or "Insert", `amVim.startInInsertMode` with type `boolean` should be easier to understand.